lib/hostinfo.cpp

```diff
@@ -75,7 +75,9 @@ void HOST_INFO::clear_host_info() {
     safe_strcpy(os_version, "");
 
 #ifdef _WIN64
+#if !defined(__aarch64__)
```

Comment:
All these are not required since WSL is supported on ARM64 devices as well: https://learn.microsoft.com/en-us/windows/wsl/install-manual

Reply:
ok, I'm encountering a relocation error during linking... let's see if I can find a better solution